notion/assets/dashboards/notion_events.json

```diff
@@ -0,0 +1 @@
+{"title":"Notion Dashboard","description":"A started dashboard for Notion page events","widgets":[{"id":6237632569622898,"definition":{"type":"free_text","text":"Page Events","color":"#000","font_size":"auto","text_align":"left"},"layout":{"x":0,"y":0,"width":12,"height":1}},{"id":2926389996605492,"definition":{"title":"Pages Created - past week","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{"live_span":"1w"},"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"data_source":"logs","name":"query1","indexes":["*"],"compute":{"aggregation":"count"},"group_by":[],"search":{"query":"@event.type:page.created"},"storage":"hot"}],"response_format":"timeseries","style":{"palette":"blue","line_type":"solid","line_width":"thick"},"display_type":"line"}]},"layout":{"x":0,"y":1,"width":4,"height":2}}],"template_variables":[],"layout_type":"ordered","notify_list":[],"reflow_type":"fixed"}
```

Comment:
As discussed in yesterday's meeting, a few key updates to this dashboard are required to ship for EAP:
- A logo
-  A text box describing what the data on the dashboard represents
- A description describing the dashboard and links to any relevant docs on the Notion side